postgres/src/main/scala/zio/sql/postgresql/PostgresModule.scala

```diff
@@ -29,10 +42,11 @@ trait PostgresModule extends Jdbc { self =>
       (year, month, day, hour, minute, second, timeZone)
 
     //Based upon https://github.com/tminglei/slick-pg/blob/master/src/main/scala/com/github/tminglei/slickpg/PgDateSupport.scala
+    //The postgres make_interval function allows for specifying weeks, but that's not supported in the PGInterval class, so we
```

Comment:
upon investigation it appears that weeks is not supported in the PGInterval class, so even I've dropped it since we wouldn't be able to handle it at the moment. It might make sense to actually remove the PG compile time dependency and just implement a string parser to convert into our internal interval class. If you like that idea I can make a separate issue and implement it.

Reply:
I do like this idea. Please create that issue and delete this comment (not the attribution though!).

Reply:
Sweet! I've created https://github.com/zio/zio-sql/issues/369 and can pick that up next.